Bucketeer/Sources/Public/BKTError.swift

```diff
@@ -53,11 +59,17 @@ extension BKTError {
                     self = .badRequest(message: errorResponse?.error.message ?? "BadRequest error")
                 case 401:
                     self = .unauthorized(message: errorResponse?.error.message ?? "Unauthorized error")
+                case 403:
+                    self = .forbidden(message: errorResponse?.error.message ?? "Forbidden error")
                 case 404:
                     self = .featureNotFound(message: errorResponse?.error.message ?? "NotFound error")
                 case 405:
                     self = .invalidHttpMethod(message: errorResponse?.error.message ?? "MethodNotAllowed error")
-                case 500:
+                case 409:
```

Comment:
The code is 499 for this error.

Reply:
Sorry, I mistook the status code. https://github.com/ca-dp/bucketeer-ios-sdk/pull/92/commits/f4ed13fe22456fbe58dc438449841f4398f86afb